doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -357,6 +364,12 @@ nRF Desktop
 
     This change in the nRF54L10 partition map is a breaking change and cannot be performed using DFU.
     As a result, the DFU procedure will fail if you attempt to upgrade the sample firmware based on one of the |NCS| v3.0 releases.
+  * The behavior of the :ref:`nrf_desktop_usb_state_pm` on USB cable disconnection.
+    While disconnecting the USB cable, the :c:enum:`USB_STATE_SUSPENDED` USB state might be reported before the :c:enum:`USB_STATE_DISCONNECTED` USB state.
+    For application to behave consistently regardless of whether the :c:enum:`USB_STATE_SUSPENDED` USB state was reported, the module submits a :c:struct:`force_power_down_event` to force a quick power down.
+    The module also restricts the power down level to the :c:enum:`POWER_MANAGER_LEVEL_SUSPENDED`.
+    Then, after the :ref:`CONFIG_DESKTOP_USB_PM_RESTRICT_REMOVE_DELAY_MS <config_desktop_app_options>` configurable delay, the module removes the power down level restriction.
+    This is done to allow you to take actions, such as restart Bluetooth LE advertising, after disconnecting the USB cable without going through reboot.
```

Comment:
These lines might also be a bit redundant as this is all included in the module documentation.

Reply:
Same here, the app behavior changes. We can wait for other reviewer to give his opinion.